packages/cascara/src/ui/Form/Form.js

```diff
@@ -106,7 +109,9 @@ const formFields = (display, data) => {
     const { myProps: moduleProps } = unhandledProps(rest, Module.propTypes);
 
     return Module ? (
-      <Module {...moduleProps} key={key} label={label} value={moduleValue} />
+      <LabeledModule attribute={attribute} isLabeled label={label}>
+        <Module {...moduleProps} key={key} label={label} value={moduleValue} />{' '}
+      </LabeledModule>
```

Comment:
@lordpixel on one hand, I like that this is more dry because the logic exists in one place, but this logic is part of the module itself, not part of Form. We are placing this wrapper here instead of in the modules themselves. This means every place we ever need to use a module, we would always have to define this wrapper for module logic. This probably significantly breaks the docs because the label logic is now no longer part of the modules and will not even show the label on those modules.

Reply:
@lordpixel maybe we call this ModuleBase instead of LabeledModule. Any reusable logic for all modules could go on this. Just a thought, but I am pretty sure we will have more boilerplate logic that can go in one place and it does not have to be about only the label.

Reply:
You actually bring a good point, @brianespinosa, I was thinking the other way around interms of re-usability but I agree this belongs more to a module rather than to the Form.

Working on it, 👍

Reply:
@brianespinosa the component was renamed to `BaseModule` and the implementation was moved to the modules.